Week3/homework/step3.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  base = a => a + 6;
+  return base;
```

Comment:
The `createBase` function should accept any base (not just 6). The output from the code below will be 16 and not 20 as I would expect.

```js
const addTen = createBase(10);
console.log(addTen(10)); 
```